# i18n-proposal: ./src/creatures/players/components/player_badge.cpp
# Generated: 2026-02-15T19:30:43.191540Z
# Category: cpp
# Transforms: 3

--- ./src/creatures/players/components/player_badge.cpp:88
-	g_logger().debug("[{}] - Loading unlocked badges: {}", __FUNCTION__, unlockedBadges.size());
+	g_logger().debug(i18n::g_translator().get("cpp.player_badge.loading_unlocked_badges", player->getLocale()), __FUNCTION__, unlockedBadges.size());
# key: cpp.player_badge.loading_unlocked_badges
# en: [{}] - Loading unlocked badges: {}

--- ./src/creatures/players/components/player_badge.cpp:92
-			g_logger().error("[{}] - Badge {} not found.", __FUNCTION__, badgeName);
+			g_logger().error(i18n::g_translator().get("cpp.player_badge.badge_not_found", player->getLocale()), __FUNCTION__, badgeName);
# key: cpp.player_badge.badge_not_found
# en: [{}] - Badge {} not found.

--- ./src/creatures/players/components/player_badge.cpp:96
-		g_logger().debug("[{}] - Badge {} found for player {}.", __FUNCTION__, badge.m_name, m_player.getName());
+		g_logger().debug(i18n::g_translator().get("cpp.player_badge.badge_found_for_player", player->getLocale()), __FUNCTION__, badge.m_name, m_player.getName());
# key: cpp.player_badge.badge_found_for_player
# en: [{}] - Badge {} found for player {}.

